examples/extensions-collection/sar-envisat-example/sar-envisat-example.json

```diff
@@ -0,0 +1,101 @@
+{
+    "type": "Feature",
```

Comment:
SAR is not one of the stable extensions and thus I think it should be only in the extension, not here.

Reply:
I do think it'd be valuable to have an extension that show referencing a 'non-core' extension. But I'm fine to remove this and then add something back int.